packages/ui/src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,57 @@
+import { ChevronDownIcon } from '@heroicons/react/24/solid';
+import type { PropsWithChildren, ReactNode } from 'react';
+import { twMerge } from 'tailwind-merge';
+
+import {
+    DropdownMenu,
+    DropdownMenuContent,
+    DropdownMenuGroup,
+    DropdownMenuItem,
+    DropdownMenuLabel,
+    DropdownMenuSeparator,
+    DropdownMenuTrigger,
+} from '@/shadcn/components/ui/dropdown-menu';
```

Comment:
I don't quite get why there is an alias to `@/shadcn` that points to components that are using another library (radix-ui)? This makes things quite confusing IMO 🤔

Reply:
A part of the requirement for this library is for things like dropdown and other similar components, we don't implement from scratch but rather on top of other library and we decided to go with `shadcn`

Reply:
I don't see a value in this, honestly. We're already pulling a UI library (Radix UI), and I think we're competent enough to create our own components. If anything, we can simply copy-paste the initial code (which we'll customize anyway) if we need to.

Reply:
Dropped shadcn